easybuild/easyconfigs/c/Clp/Clp-1.17.6-foss-2021a.eb

```
@@ -0,0 +1,61 @@
+easyblock = 'ConfigureMake'
+
+name = 'Clp'
+version = '1.17.6'
+
+homepage = "https://github.com/coin-or/Clp"
+description = """Clp (Coin-or linear programming) is an open-source linear programming solver.
+It is primarily meant to be used as a callable library, but a basic,
+stand-alone executable version is also available."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://www.coin-or.org/download/source/%(name)s/']
+sources = [SOURCE_TGZ]
+checksums = ['f9cfcdd3db23bbc2d18b562012a7e25b99a40e8ce72dc296bd46dddb2970e12a']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('Doxygen', '1.9.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('METIS', '5.1.0'),
+    ('MUMPS', '5.4.0', '-metis'),
+    ('CoinUtils', '2.11.4'),
+    ('Osi', '0.108.6'),
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+]
+
+# Use BLAS/LAPACK from OpenBLAS
+configopts = '--with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK" '
+
+# Use METIS AND MUMPS from EB
+# --with-metis-lib is ignored
+configopts += '--with-metis-lib="-lmetis" '
+configopts += '--with-mumps-lib="-lesmumps -lcmumps -ldmumps -lsmumps -lzmumps -lmumps_common -lpord -lmpi_mpifh '
+configopts += '-lmetis  -lscotch -lptscotch -lptscotcherr -lscotcherrexit -lscotcherr -lscalapack" '
+
+# Disable GLPK because Clp requires headers from its sources
+configopts += '--without-glpk '
+
+# Use CoinUtils from EB
+configopts += '--with-coinutils-lib="-lCoinUtils" '
+configopts += '--with-coinutils-datadir=$EBROOTCOINUTILS/share/coin/Data'
+
+# Use Osi from EB
+configopts += '--with-osi-lib="-lOsi" '
+configopts += '--with-osi-datadir=$EBROOTOSI/share/coin/Data '
+
+sanity_check_paths = {
+    'files': ['bin/clp'] + ['lib/lib%s.%s' % (lib, SHLIB_EXT) for lib in ['Clp', 'ClpSolver', 'OsiClp']],
```

Variables in list comprehensions should be single-letter (except for `l`, because it's easy to confuse `l` with `I`)  or something like `local_lib`.
Here, I'd use `x`:

```suggestion
    'files': ['bin/clp'] + ['lib/lib%s.%s' % (x, SHLIB_EXT) for x in ['Clp', 'ClpSolver', 'OsiClp']],